commitlint.config.js

```diff
@@ -1,8 +1,24 @@
-// @noflow
+const fs = require("fs");
+
+function fetchComponentDirs() {
```

Comment:
instead of running only on parent folders, try to use globby, to include also children folders, for instance, if i made a change inside `ItinerarySegmentStop`, I want to have that scope allowed too :)

Reply:
Good point. I did not use `globby`, but its original form `fast-glob` instead. But the end result is achieved 🙂

Reply:
hm, it's ok, but we have globby already as a dependency from `zx` :) so that's why I suggested that, no need for additional dependency :)

Reply:
We also have `fast-glob` 😄 you see that yarn.lock did not change when I added as a devDep, so I think that's pretty much the same.

Plus, I was not able to use `globby` with `require` and so, to avoid more changes to the file, I proceeded with `fast-glob`

Reply:
np :) LGTM